packages/@tests/smoke/node/src/index.ts

```diff
@@ -1,4 +1,22 @@
-import '@fluencelabs/js-client';
-import { runTest } from '@test/aqua_for_test';
+/**
+ * Copyright 2023 Fluence Labs Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
 
-runTest().then(() => console.log('Smoke tests succeed!'));
+import "@fluencelabs/js-client";
+import { runTest } from "@test/aqua_for_test";
+
+await runTest().then(() => {
+  console.log("Smoke tests succeed!");
+});
```

Comment:
you don't need `then` if top level await works
```suggestion
await runTest()
console.log("Smoke tests succeed!");
```